<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="KARTIK-SHARMA" failures="0" tests="4" name="com.tests.LoginWithEmail" time="0.019" errors="1" timestamp="2025-02-13T13:25:25 IST" skipped="3">
  <testcase classname="com.tests.LoginWithEmail" name="AppActions" time="0.019">
    <error message="Cannot invoke &quot;io.appium.java_client.android.AndroidDriver.findElement(org.openqa.selenium.By)&quot; because &quot;this.driver&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "io.appium.java_client.android.AndroidDriver.findElement(org.openqa.selenium.By)" because "this.driver" is null
at com.pageobjects.LoginPage.<init>(LoginPage.java:19)
at com.main.Capablities.AppActions(Capablities.java:94)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:655)
at org.testng.TestRunner.beforeRun(TestRunner.java:644)
at org.testng.TestRunner.run(TestRunner.java:610)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- AppActions -->
  <testcase classname="com.tests.LoginWithEmail" name="navigateToLogin" time="0.000">
    <skipped/>
  </testcase> <!-- navigateToLogin -->
  <testcase classname="com.tests.LoginWithEmail" name="verifyValidEmail" time="0.000">
    <skipped/>
  </testcase> <!-- verifyValidEmail -->
  <testcase classname="com.tests.LoginWithEmail" name="retrieveAndEnterOTP" time="0.000">
    <skipped/>
  </testcase> <!-- retrieveAndEnterOTP -->
</testsuite> <!-- com.tests.LoginWithEmail -->
